cmd/helm/package.go

```diff
@@ -146,7 +150,10 @@ func (p *packageCmd) run(cmd *cobra.Command, args []string) error {
 
 	name, err := chartutil.Save(ch, dest)
 	if err == nil {
-		debug("Saved %s to current directory\n", name)
+		debug("Saved into: %s\n", name)
+		fmt.Fprintf(p.out, "Saved into: %s\n", name)
```

Comment:
How come you're using the same statement for debug as you are for printing out?
In this case, why would you not run the command with the `--debug` flag?
This is what the output looks like right now with the `--debug` flag:

```
helm [FengyunPan-fix-package*]$ bin/helm package docs/examples/nginx/ --debug
[debug] Saved into: /Users/michellenoorali/gocode/src/k8s.io/helm/nginx-0.1.0.tgz

Saved into: /Users/michellenoorali/gocode/src/k8s.io/helm/nginx-0.1.0.tgz
[debug] Saved /Users/michellenoorali/gocode/src/k8s.io/helm/nginx-0.1.0.tgz to /Users/michellenoorali/.helm/repository/local

Saved /Users/michellenoorali/gocode/src/k8s.io/helm/nginx-0.1.0.tgz to /Users/michellenoorali/.helm/repository/local
helm [FengyunPan-fix-package*]$
```

As you can see we're just printing out things twice now.